xep-0333.xml

```diff
@@ -236,6 +242,48 @@
   <section2 topic='Interaction with Chat States' anchor='chatstates'>
     <p>Chat Markers MAY be used alongside Chat States.</p>
   </section2>
+  <section2 topic='Usage within MUC' anchor='rules-muc'>
+    <p>Markers may be used within a MUC to indicate read status of each occupant.</p>
+    <p>Within the context of a MUC messages are relayed through the MUC's own JID. In a
+       MUC that preserves the 'id' attribute chosen by the sender of the message this
+       'id' attribute cannot be considered unique, as it may be unintentionally or
+       even maliciously reused by another MUC occupant.</p>
+
+    <p>Therefore, if a &xep0359; &lt;stanza-id&gt; element is present within a markable stanza,
+       with a 'by' attribute matching the MUC's bare JID, this is the id that MUST
+       be used when returning chat marker updates.</p>
+
+<example caption='Example MUC message with a markable Chat Marker'>
+<![CDATA[
+<message
+    from='coven@chat.shakespeare.lit/firstwitch'
+    to='coven@chat.shakespeare.lit'
+    id='message-1'
+    type='groupchat'>
+  <thread>Act IV, Scene I</thread>
+  <body>Thrice the brinded cat hath mew'd.</body>
+  <markable xmlns='urn:xmpp:chat-markers:0' />
+  <stanza-id xmlns='urn:xmpp:sid:0' by='coven@chat.shakespeare.lit' id='39K7ZYIp' />
+</message>
+]]>
+</example>
+
+<example caption='Example marker response to a markable MUC message'>
+<![CDATA[
+<message
+    from='coven@chat.shakespeare.lit/secondwitch'
+    to='coven@chat.shakespeare.lit'
+    id='message-2'
+    type='groupchat'
+    >
```

Comment:
```suggestion
    type='groupchat'>
```

This fits better to the formatting of the other examples in other XEPs.

Reply:
Fixed, thanks!